cmd/tracee-ebpf/main.go

```diff
@@ -176,6 +176,8 @@ func main() {
 			cfg.ChanEvents = make(chan trace.Event)
 			cfg.ChanErrors = make(chan error)
 
+			cfg.RequiredInits = &tracee.RequiredInitConfig{}
```

Comment:
I don't understand why do we need that for.
If an event requires a kconfig, isn't it enough to declare it in its dependencies?

Reply:
We need it to determine for the whole program if it should load the kallsyms file or not.
I created a category in the configuration for things that require initialization if are required by the code, so when the question if to initialize them is given we have a single bool flag to determine it.